homeassistant/components/awair/translations/en.json

```diff
@@ -24,12 +27,28 @@
                 },
                 "description": "Please re-enter your Awair developer access token."
             },
-            "user": {
+            "local" :{
+                "data": {
+                    "host": "Hostname or IP address of the device"
+                },
+                "description": "Awair Local API must be enabled following these steps: https://support.getawair.com/hc/en-us/articles/360049221014-Awair-Element-Local-API-Feature"
+            },
+            "cloud" :{
                 "data": {
                     "access_token": "Access Token",
                     "email": "Email"
                 },
                 "description": "You must register for an Awair developer access token at: https://developer.getawair.com/onboard/login"
+            },
+            "discovery_confirm": {
+                "description": "Do you want to setup local {model} device: {host}?"
+            },
+            "user": {
+                "menu_options": {
+                    "cloud": "Cloud API",
+                    "local": "Local API"
+                  },
+                "description": "Awair provides two APIs, a Cloud API with data summarized over 5 minutes, and a beta Local API which can be polled for real time data."
```

Comment:
We should steer everyone to local. It's local and faster. Cloud should only be used for devices not on the local LAN.

So we should make the description represent that.

Pick local for the best experience. Only use cloud if the device is not connected to the same network as Home Assistant.